### ISSUE TITLE
User should get "Build success" message when build is successful.

### ISSUE THREAD

**Issue by v-pavanp** [Microsoft/vscode#5179]
- VSCode Version: 1.0.0-alpha
- OS Version: Windows10

Steps to Reproduce:
1. Launch VS code and open any folder which contains .ts file.
2. Configure tasks.json and tsconfig.json files.
3. Set "showOutput": "always" setting into tasks.json.
4. Perform build operation and verify error or success message in the output window.

Actual: Getting proper message when there is any build error but not getting any message when build is successful.
Expected: When build is successful user should get the "Build success" message something similar to Visual Studio.

![image](https://cloud.githubusercontent.com/assets/17735796/14449052/ebf2cf3c-0023-11e6-96f4-1f234a02eeed.png)

**Comment by dyong0** [comment]
@egamma Can I make a pull request for this issue? I'd like to confirm it before starting dev. https://github.com/Microsoft/vscode/wiki/How-to-Contribute#where-to-contribute

**Comment by egamma** [comment]
@dyong0 thanks for offering a PR, the timing is not ideal given that we are making some significant changes to the tasks infrastruture. @dbaeumer what do you think?

**Comment by dbaeumer** [comment]
@dyong0 thinking about this again I am not sure that we should add something like this to the task framework. The task framework calls into an external build system (not like VS) and therefore doesn't know what the external process prints to the console. This task system could already have printed ('Build successfull'). So how would we avoid double printing. IMO we should rely on the build system we call into to properly print messages.

We indicate the start and stop of a build using a spinner in the status bar. So IMO we should invest into making this a better UI feedback than printing additional stuff to the console.

As @egamma said, we are moving the task running from the output panel to the terminal. Until this is done I think accepting PR will be difficult. The plan is to make the terminal the default for the next release.

**Comment by dyong0** [comment]
@dbaeumer I entirely agree with your opinion. But I think that @v-pavanp wanted to see a message when the task gets finished, so that he notices it. As you said an external build system will show output, however currently it's almost like running a shell script which runs a build system. I hope that Code gives us a better UX by making a noticeable notification when a build task gets finished.

And also regarding the plan to moving into the terminal, I'm very happy with it. That would make users focus on the editor and terminal. No output panel is required. Thank you for your contributions.

**Comment by alexr00** [comment]
Duplicate of #7090, since that covers all post task actions.
